src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,48 @@
+package racingcar;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+	@Test
+	@DisplayName("Car이 생성된 후 초기 위치는 0이다.")
+	void whenCarCreatedThenPositionIs0() {
+		Car car = new Car();
+		assertThat(car.getPosition()).isEqualTo(0);
+	}
+
+	@Test
+	@DisplayName("Car가 움직이면 위치는 증가한다.")
+	void whenCarMoveThenPositionIncreases() {
+		Car car = new Car();
+		car.move();
+
+		assertThat(car.getPosition()).isGreaterThan(0);
+	}
+
+	@Test
+	@DisplayName("움직일 수 있는 조건을 만족하면 위치는 증가한다.")
+	void increasePositionWithSatisfiedStrategy() {
+		CarMoveStrategy carMoveStrategy = new TestCarMoveStrategy(true);
+		assertThat(carMoveStrategy.isMovable()).isTrue();
+
+		Car car = new Car(carMoveStrategy);
```

Comment:
메소드가 하나인 인터페이스의 장점을 살리면 아래와 같은 표현도 가능합니다!
```suggestion
		Car car = new Car(() -> true);
```

Reply:
ㅇ0ㅇ.. 부끄럽지만 위 표현은 몰랐네요..! 감사합니다 😄